EIPS/eips-chain-address-uri-schema

```diff
@@ -0,0 +1,68 @@
+
+---
+eip: <to be assigned>
+title: Chain URI Format for Human-Readable Blockchain Transactions
+description: A URI scheme that encodes address, token, and chain stack in a human-readable and linkable format.
+author: Dale Clifford (@daleclifford)
+discussions-to: https://github.com/daleclifford/eips/discussions
+status: Draft
+type: Standards Track
+category: Interface
+created: 2025-05-01
+requires: 
+---
+
+## Abstract
+
+This proposal introduces a standardized URI format using the `chain:` scheme to enable human-readable, machine-actionable blockchain interactions. The format builds on familiar internet conventions (like email and `mailto:` URIs) to make blockchain transactions clearer and safer for end users. It supports tokens, layered chain networks, and optional query parameters like amount and memo.
+
+## Motivation
+
+Blockchain addresses and transaction routing are opaque and error-prone for most users. The average person does not understand hexadecimal wallet addresses, chain compatibility, or token contract safety. This proposal provides a clear, readable, and linkable format to express intent, destination, and token context—bringing blockchain UX in line with the usability of email, links, and QR codes.
+
+## Specification
+
+### URI Format
+
+```
+chain:<address>@$<token>.<chain_path>?<parameters>
+```
+
+### Components
+
+- `address`: The full public blockchain address (e.g. 0x1234abcd...)
+- `$token`: Token being sent, using a dollar-prefixed symbol (e.g. $usdc)
+- `chain_path`: Dot-separated hierarchy (e.g. worldchain.op.eth)
+- `parameters`: Optional key-value query string (e.g. amount, memo)
```

Comment:
Before I assign a number, I'd recommend that you flesh out this section a bit further. It seems very close to https://eips.ethereum.org/EIPS/eip-681 so far.